tests/src/common/WskCli.java

```diff
@@ -838,14 +852,16 @@ private boolean logsForActionContain(String action, String sRegex, int initialWa
      *            name of the action.
      * @param w
      *            the string to search for
+     * @param since
+     * 			  since parameter for cli. Only activations after this timestamp will be returned.
```

Comment:
scan code checks for leading tabs, not ones since a comment (or within a line) @psuter FYI.